compiler/resolution/functionResolution.cpp

```diff
@@ -8249,6 +8253,53 @@ static void cleanupAfterRemoves() {
 }
 
 
+static void printUnusedFunctions() {
+#ifdef PRINT_UNUSED_FNS_TO_FILE
```

Comment:
Was this defined in an earlier commit, or is it just debugging support that got left behind?

Reply:
It's useful for running the whole test suite and saving the results to files so we can see what functions are not used by the test suite.  Just define this macro (and maybe also `EXIT_AFTER_PRINTING_UNUSED_FNS`) and run paratest.

Reply:
Would you mind documenting that in the developers documentation or something, then?  (and maybe add a comment pointing to that explanation of its use)  It seems like it would get overlooked otherwise

Reply:
I added some comments about these macros at the top of `printUnusedFunctions`.  Do you think it needs documentation beyond that?  If yes, where would you recommend putting it?